### PR TITLE
Import all available libraries in --repl mode

### PR DESCRIPTION
**Pull request by JaroslavTulach**
### Pull Request Description

Continuation of REPL work (#10709): Import all available libraries when running `--repl`.

### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [x] The documentation has been updated, if necessary.
- [x] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
- [x] Unit tests have been written where possible.

### CODE REVIEW

**Comment by JaroslavTulach** [comment]
These are the includes that `--repl` currently generates:
```ruby
import Standard.Base.Runtime.Debug
from Standard.Base import all
from Standard.Test import all
from Standard.Visualization import all
from Standard.Geo import all
from Standard.Microsoft import all
from Standard.Examples import all
from Standard.Google_Api import all
from Standard.Snowflake import all
from Standard.Table import all
from Standard.Searcher import all
from Standard.AWS import all
from Standard.Image import all
from Standard.Database import all
```

File: engine/runner/src/main/java/org/enso/runner/Main.java [hunk 1]
```diff
@@ -933,7 +925,18 @@ private void runRepl(
...
                 .enableStaticAnalysis(enableStaticAnalysis)
                 .build());
-    var mainModule = context.evalModule(dummySourceToTriggerRepl, replModuleName);
+
+    var sb = new StringBuilder();
+    sb.append("import Standard.Base.Runtime.Debug\n");
+
+    for (var libName : context.getTopScope().getLibraries()) {
+      sb.append("from ").append(libName).append(" import all\n");
+    }
```
Comment (radeusgd) [inline engine/runner/src/main/java/org/enso/runner/Main.java]: won't we risk having clashing symbols in the libraries?
We probably shouldn't have but I don't think we have any test that ensures this.

Alternatively, we could do `import X.Y` - then the user would need to use FQN to access the imported symbols. What do you think?

**Review by radeusgd — APPROVED** [review]
Awesome :D

File: engine/runner/src/main/java/org/enso/runner/Main.java (hunk 1 above)
Comment (JaroslavTulach) [reply engine/runner/src/main/java/org/enso/runner/Main.java]: We don't have any conflicts right now, but you are the user...

Comment (radeusgd) [reply engine/runner/src/main/java/org/enso/runner/Main.java]: I think `import all` is more user-friendly, so long as we test somewhere that we don't introduce conflicts.

Comment (JaroslavTulach) [reply engine/runner/src/main/java/org/enso/runner/Main.java]: Test to verify script compiles without conflicts provided in 78db664

Comment (radeusgd) [reply engine/runner/src/main/java/org/enso/runner/Main.java]: Thank you

**Review by Akirathan — APPROVED** [review]
Nice!

**Review by 4e6 — APPROVED** [review]

**Review by hubertp — APPROVED** [review]

**Pull request merged**